latexify/core.py

```diff
@@ -36,11 +36,15 @@ def generic_visit(self, node):
   def visit_Module(self, node):
     return self.visit(node.body[0])
 
-  def visit_FunctionDef(self, node):
+  def visit_FunctionDef(self, node, equal:str=r'\triangleq'):
```

Comment:
for "equal": 
the fact is that I haven't decided how to implement this meanwhile keep the current interface work (maybe allowing @with_latex(equal="\equal") or something like that will be a good idea). I add this only because I seldom use "\triangleq", and I feel it necessary to have "\equal" for some occasions.

Reply:
I will try to make this work in the next commit.

Reply:
I meant this option could *never* be used by users: all `visit_***` functions are called internally by `NodeVisitor` with only `node` parameter. We have basically no way to pass extra arguments.

Reply:
yes I made this mistake because I've just started learning ast and am not so familiar with its logic. Will remove it.
But I still feel it necessary to leave interface for users to change the form of equal mark. If latexify aims to work on the most cases, and "keep minimality about extra conventions"-I strongly agree with this-should be made into practice, then latexify should leave the most freedom for users to customize their output.
If you also agree with this, I will try to find my approach to implement this with no original interface changed.

Reply:
I have suggested to add a similar option like `math_symbol`. It should work and not affect the behavior of existing usages.

Reply:
OK. I take that as the final result of discussion and will try to implement it in the next commit.